airbyte-integrations/connectors/source-gitlab/metadata.yaml

```diff
@@ -4,13 +4,13 @@ data:
     sl: 200
   allowedHosts:
     hosts:
-      - "*"
+      - ${api_url}
```

Comment:
I believe this will not allow OSS users to set up connection to a self hosted Gitlab instance

Reply:
why? if user uses self-hosted gtlab instance, he can pass his own api_url, and in this case api url will be allowed.

Reply:
You are right. Sorry, overlooked it